### PR TITLE
add commonwealth tvl

### PR DESCRIPTION
**Pull request by samswealth**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
Common Wealth

##### Twitter Link:
https://x.com/joincommonwlth

##### Website Link:
http://app.joincommonwlth.xyz/

##### Logo (High resolution, will be shown with rounded borders):
common

##### Current TVL:
2.2M base

##### Chain:
Base (EVM)

##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)
common-wealth

##### Short Description (to be shown on DefiLlama):
Early-stage VC access for the 99%. Fully decentralised. Fully on-chain. The way it was meant to be.

##### Token address and ticker if any:
0x99b2b1a2adb02b38222adcd057783d7e5d1fcc7d

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Launchpad

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/commonwealth exports TVL: 
        
 
 
    base-staking              11.37 M
    staking                   11.37 M
    base                      2.66 M
    
    total                    2.66 M

**Comment by llamatester** [comment]
The adapter at projects/commonwealth exports TVL: 
        
 
 
    base-staking              2.75 M
    staking                   2.75 M
    base                      0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/treasury/commonwealth.js exports TVL: 
        
 
 
    base                      2.66 M
    base-ownTokens            997.01 k
    ownTokens                 997.01 k
    
    total                    2.66 M

File: projects/commonwealth/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const { staking } = require('../helper/staking');
+
+const TREASURY = [
+  '0xdE70B8BC5215BdF03f839BB8cD0F639D4E3E2881',
```
Comment (g1nt0ki) [inline projects/commonwealth/index.js:4]: hi @samswealth thanks for the PR; we dont track vesting atm. This treasury is project treasury or user funds that they can withdraw at any point?

Comment (samswealth) [reply projects/commonwealth/index.js:4]: hi @g1nt0ki, project treasury.

**Pull request merged**